docs/src/pages/_app.page.tsx

```diff
@@ -1,7 +1,5 @@
 import Head from 'next/head';
-import '../styles/styles.css';
-import '@aws-amplify/ui-react/styles.css';
-import '../styles/primitivesStyles.css';
+import '../styles/index.css';
```

Comment:
@ericclemmons NextJS was including these styles in incorrect order, breaking the docs example CSS. Changed this to a single CSS import and handled ordering of other imports there.